test/core/f32.wast

```diff
@@ -2531,3 +2531,13 @@
 (assert_invalid (module (func (result f32) (f32.nearest (i64.const 0)))) "type mismatch")
 (assert_invalid (module (func (result f32) (f32.sqrt (i64.const 0)))) "type mismatch")
 (assert_invalid (module (func (result f32) (f32.trunc (i64.const 0)))) "type mismatch")
+
+;; These float literal patterns are only allowed in scripts, not in text format.
+(assert_malformed
+  (module quote "(func (result f32) (f32.const nan:arithmetic))")
+  "unexpected token"
+)
+(assert_malformed
+  (module quote "(func (result f32) (f32.const nan:canonical))")
+  "unexpected token"
+)
```

Comment:
Could you add a similar test to f64.wast that tests the same thing for the f64 type, to keep these two files in sync?

Reply:
Oh yea good suggestion, done!